src/hera/workflows/script.py

```diff
@@ -224,4 +223,18 @@ def _get_parameters_from_callable(source: Callable) -> Optional[List[Parameter]]
     return [Parameter(name=n, default=v) for n, v in source_signature.items()]
 
 
-__all__ = ["Script"]
+def script(**script_kwargs):
+    def script_wrapper(func):
+        s = Script(name=func.__name__.replace("_", "-"), source=func, **script_kwargs)
+
+        def task_wrapper(**task_params):
+            return s.__call__(**task_params)
+
+        return task_wrapper
+
+    return script_wrapper
+
+
+task = script
```

Comment:
I would prefer if we didn't name script as task here. Might confuse users and might cause a hindrance if in the future we do want to add tasks of other types. It also doesn't match with the step type where it can also be used.